src/GlobalData.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+"use strict";
+
+/**
+ * Populate and export global data
+ */
+
+import { ActionBmwCodeCell } from "./ActionBmwCodeCell";
+
+export const adaptivePaletteGlobals = {
+  // The map between the BCI-AV IDs and the code consumed by the Bliss SVG
+  // builder.  The map itself is set asynchronously.
+  blissaryIdMapUrl: "https://raw.githubusercontent.com/hlridge/Bliss-Blissary-BCI-ID-Map/main/blissary_to_bci_mapping.json",
+  blissaryIdMap: null
+};
+
+// For debugging
```

Comment:
Please remove the debugging line before the merge. Thanks.

Reply:
Can you clarify?  Do you mean remove just the comment line, "`// For debugging`"?  Or, the variable `fetchCount` and the `console.debug()` where the `fetchCount` value is logged?

I have checked the fetch count periodically and only once was it greater than one, which actually led to uncovering a bug.  But since then, it is always been "1".  I'm tempted to remove it at this point.

Reply:
I meant to remove the debugging of the fetchCount completely if the code is stable.